### ISSUE TITLE
Improve split functions

### ISSUE THREAD

**Issue by PeopleInside** [flagrow/split#18]
Hi, i love this extension and believe is very useful but seems are missing some useful functions.

Seems right now you can split a message in the thread to a new thread.

I believe will be nice to be able to split a post to another existent post too also with different tags, if this is not possible directly from the first thread will be useful split the post in a new thread than be able in this thread to use again the split function that now is not showed in a first post for split into an existing thread with different tags.

Actually seems you can split a message only in a new thread not in existent discussion, will be nice to be able to do this, in future.

**Comment by luceos** [comment]
Yes there's plenty of room for improvement. Most extensions at this point are still very simplistic in their feature set.

**Comment by Megalomaniak** [comment]
> Seems right now you can split a message in the thread to a new thread.

I find it very counter intuitive in how even that works. Rather than picking the split from here menu option and then prompting you for a new topics name the post just remains hanging where it is and you have to now press on the 'split to here' link for it to finally prompt you for a new topics name. Very confusing.

**Comment by luceos** [comment]
@Megalomaniak perhaps the move extension is more helpful to you; https://extiverse.com/extension/sycho/flarum-move-posts

**Comment by Megalomaniak** [comment]
@luceos Hmm, perhaps but I'd still expect split to work a bit better in UX terms. I'm really not sure what the thinking behind the current split UX was.

I'd understand the current design if the idea was that the 'split to here' would also appear in other topics if you navigate out the progenitor topic and into the one you want to split the post into, but that's clearly not how it works.

**Comment by luceos** [comment]
>  @luceos Hmm, perhaps but I'd still expect split to work a bit better in UX terms. I'm really not sure what the thinking behind the current split UX was.
> 
> I'd understand the current design if the idea was that the 'split to here' would also appear in other topics if you navigate out the progenitor topic and into the one you want to split the post into, but that's clearly not how it works.

The idea of split is to split a serie of posts from the current discussion. What you do with it afterwards is another thing. At discuss we used it intensively with the discussion merge extension.
